cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -86,87 +85,97 @@ Kitematic and then download, start, and connect to an instance of CDAP.
 Docker from a Command Line
 ==========================
 
-- Docker is available for a variety of platforms. Download and install Docker in your environment by
-  following the `platform-specific installation instructions <https://docs.docker.com/engine/installation/>`__
-  from `Docker.com <https://docker.com>`__ to verify that Docker is working and has
-  started correctly.
-  
-  If you are not running on Linux, you need to start the Docker Virtual Machine (VM) before you
-  can use containers. For example:
+Docker is available for a variety of platforms. Download and install Docker in your
```

Comment:
Yes, Docker kind of skates over this detail by referring to it as a "Docker environment".